50.mender-binary-delta/docs.md

```diff
@@ -7,6 +7,73 @@ shortcode-core:
 github: false
 ---
 
+## mender-binary-delta 1.3.0
+
+_Released 09.28.2021_
+
+### Statistics
+
+A total of 602 lines added, 218 removed (delta 384)
+
+| Developers with the most changesets | |
+|---|---|
+| Ole Petter Orhagen | 8 (33.3%) |
+| Kristian Amlie | 8 (33.3%) |
+| Lluis Campos | 6 (25.0%) |
+| Alex Miliukov | 1 (4.2%) |
+| Manuel Zedel | 1 (4.2%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Kristian Amlie | 389 (62.6%) |
+| Ole Petter Orhagen | 175 (28.2%) |
+| Lluis Campos | 41 (6.6%) |
+| Manuel Zedel | 13 (2.1%) |
+| Alex Miliukov | 3 (0.5%) |
+
+| Developers with the most lines removed | |
+|---|---|
+| Manuel Zedel | 11 (5.0%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 24 (100.0%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 621 (100.0%) |
+
+| Employers with the most hackers (total 5) | |
+|---|---|
+| Northern.tech | 5 (100.0%) |
```

Comment:
We haven't usually published statistics for closed components, since they are not visible anyway. And I suppose it's a bit of self-bragging, since no one else can be on the list. No strong feelings against it though, I'll let you decide.

Reply:
I didn't realize. I'll remove them